pkgs/applications/misc/fff/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, lib, fetchFromGitHub, makeWrapper, bash, xdg_utils, file, coreutils }:
+
+with lib;
+
+stdenv.mkDerivation rec {
+  name = "fff";
+  version = "1.5";
+
+  src = fetchFromGitHub {
+    owner = "dylanaraps";
+    repo = name;
+    rev = version;
+    sha256 = "0jvv9mwj0qw3rmg1f17wbvx9fl5kxzmkp6j1113l3a6w1na83js0";
+  };
+
+  pathAdd = makeSearchPath "bin" [ xdg_utils file coreutils ];
+  buildInputs = [ makeWrapper ];
+
+  installPhase = ''
+    mkdir -p $out/{bin,share/man/man1,share/doc/fff}
+    mv ${name} $out/bin
```

Comment:
Please use just `fff` here instead of `${name}`.

Also
```
mkdir $out/foo
mv bar $out/foo
```
can be replaced with
```
install -D bar "$out"/foo/bar
```